### DISCUSSION TITLE
Custom Confirm Change of Email Template

### DISCUSSION THREAD

**Discussion by tourbillonlabs** [supabase/supabase#3800] [Questions] [answered]
Hello hello,

Sorry if I've missed this in the docs or tickets, but in the Supabase App -> Authentications -> Templates I see that we can customise the email templates for `Confirm Signup`, `Reset password`, `Magic Link`, and `Invite user`, but how do we customise the template for confirmation of change of email?

Thanks

**Comment by J0** [comment, marked as answer]
Hey @tourbillonlabs,

Thanks for the query! It should be exposed in the UI now:

![CleanShot 2022-09-29 at 13 22 41@2x](https://user-images.githubusercontent.com/8011761/193007024-5b70a493-0e40-4865-a239-cf9551ee8616.png)
